src/main/kotlin/no/nav/familie/ef/sak/mapper/TilkjentYtelseMapper.kt

```diff
@@ -15,3 +18,14 @@ fun TilkjentYtelse.tilDto(): TilkjentYtelseDto {
                                                         samordningsfradrag = andel.samordningsfradrag)
                              })
 }
+
+fun TilkjentYtelse.tilBeløpsperiode(): List<Beløpsperiode> {
```

Comment:
Er det sån att vi egentlige burde returnere AndelTilkjentYtelseDto ? 

Ser ikke helt hvorfor man trenger å returnere Beløpsperiode til frontend, hverken her eller i frontend.
`IBeregningsgrunnlag` inneholder flere felter som ikke er i bruk, virker som att de som er i bruk er de samme som er på AndelTilkjentYtelse

Reply:
Dvs, kan man bytte ut IBeløpsperiode i frontend og kun bruke noe som tilsvarer AndelTilkjentYtelse? 
Tenker den kommer bli brukt på flere steder og er enkel å gjennbruke, i stedet for å sette nullable på felter :P

Reply:
Jeg kan ju godkjenne denne nå så att du kan få ut funksjonaliteten, hvis vi skriver om disse delene senere? 

@alexandraback @charliemidtlyng @eirarset Trenger vi beløpsperiode i frontend? Eller er det tilsrekkelig med noe som er likt AndelTilkjentYtelse?

Reply:
Tror inte jag förstår denna settning "Ser ikke helt hvorfor man trenger å returnere Beløpsperiode til frontend, hverken her eller i frontend." Beløpsperiode brukes i `<Utregningstabell/>` på frontend? Kan fint returnera AndelTilkjentYtelseDto men då måste jag göra mappningen på frontend

Reply:
I mitt huvud är AndelTilkjentYtelse och Beløpsperiode två olika konsept. Imho hade det vart bättre att lagra Beløpsperioder och sända dom separat til iverksett (istället för att lägga till inntekt, samordningsfradrag etc på AndelTilkjentYtelse )

Reply:
Delvis enig, men tror man kan tenkte AndelTilkjentYtelse som resultatet fra beregning?

Reply:
En utfordring jeg ser er at aktivitet ikke kommer ut til frontend her?

Reply:
Borde den den det ? Siden detta er endepunktet er för å hente lagret utregning som ska vises i <Utregningstabell/> når behandlingen er låst?

Reply:
Lagret utregning, eller utregningen er ju egentligen andelene? :P

Reply:
Det er sant, det blir et eget endepunkt for å trikse frem aktivitet og historikk. Dette funker 👍